src/test/java/com/exasol/releasedroid/adapter/github/progress/ProgressFormatterTest.java

```diff
@@ -0,0 +1,242 @@
+        when(monitor.elapsed()) //
+                .thenReturn(Duration.ofMillis(300)) //
+                .thenReturn(Duration.ofSeconds(1));
+        when(monitor.remaining()).thenReturn(estimation) //
+                .thenReturn(Duration.ofSeconds(59));
+        final ProgressFormatter testee = startFormatter(monitor, estimation);
+
+        final String status1 = testee.status();
+        assertThat(status1, containsString(green("0:00:00 elapsed")));
+        assertThat(status1, containsString(yellow("~ 1 minute remaining")));
+        // sleep 1 s
+        final String status2 = testee.status();
+        assertThat(status2, containsString(green("0:00:01 elapsed")));
+        assertThat(status2, containsString(yellow("59 seconds remaining")));
+    }
+
+    private ProgressMonitor mockProgressMonitor(final Duration estimation) {
+        final ProgressMonitor monitor = Mockito.mock(ProgressMonitor.class);
+        when(monitor.getEstimation()).thenReturn(Optional.of(estimation));
+        when(monitor.eta()).thenReturn(toLocal(INSTANT));
+        return monitor;
+    }
+
+    @Test
+    void timeout() throws InterruptedException {
+        final Duration timeout = Duration.ofMillis(100);
+        final ProgressMonitor monitor = Mockito.mock(ProgressMonitor.class);
+        when(monitor.isTimeout())//
+                .thenReturn(false) //
+                .thenReturn(true);
+        final ProgressFormatter formatter = new ProgressFormatter.Builder(monitor) //
+                .timeout(timeout) //
+                .start();
+        assertThat(formatter.timeout(), is(false));
+        // sleep 200 ms
+        assertThat(formatter.timeout(), is(true));
+    }
+
+    @Test
+    void progress() throws InterruptedException {
+        final Duration estimation = Duration.ofSeconds(1);
+        final ProgressMonitor monitor = mockProgressMonitor(estimation);
+        when(monitor.elapsed()) //
+                .thenReturn(Duration.ofSeconds(0)) //
+                .thenReturn(Duration.ofSeconds(1)) //
+                .thenReturn(Duration.ofSeconds(2));
+        when(monitor.remaining()) //
+                .thenReturn(Duration.ofSeconds(1)) //
+                .thenReturn(Duration.ofSeconds(0)) //
+                .thenReturn(Duration.ofSeconds(-1));
+
+        final ProgressFormatter testee = startFormatter(monitor, estimation);
+        final String[][] expected = { //
+                { "0:00:00 elapsed", "1 second remaining", "0%", "[", ">" },
+                { "0:00:01 elapsed", "0 seconds remaining", "100%", "[===================", "|>" },
+                { "0:00:02 elapsed", "1 second overdue", "200%", "[=========", "|=========>" } };
+        for (int i = 0; i < 3; i++) {
+            verifyContents(testee.status(), expected[i]);
+        }
+    }
+
+    private void verifyContents(final String actual, final String... expected) {
+        for (final String e : expected) {
+            assertThat(actual, containsString(e));
+        }
```

Comment:
Proposal: convert list of expected values to array of matchers, then use `allOf(matchers)`